src/Avalonia.Base/Reactive/LightweightObservableBase.cs

```diff
@@ -205,4 +235,59 @@ protected virtual void Subscribed(IObserver<T> observer, bool first)
         {
         }
     }
+
+    // Uncomment this class to track observer counts
```

Comment:
Could you please remove this? While having some statistics is very nice, it's likely to stay commented forever :)

Reply:
Yep, fixed via 150c7ca